res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,444 @@
+  status -= 32;
+  var num_parameters = engine.getValue("[EffectRack1_EffectUnit" + currentDeck + "_Effect1]", "num_parameters");
+  if (num_parameters > 3) {num_parameters = 3;};
+  if (num_parameters) {
+    for (var i = 1, j = 30; i <= num_parameters; i++) {
+      Jockey3ME.effectParamLedSet(currentDeck, i, status, j);
+      j++;
+    };
+  }
+  Jockey3ME.effectMixLedSet(currentDeck, status, 29);
+  // Jockey3ME.effectSelectLedSetNumEffect(currentDeck,status,92);
+}
+
+// Browser Knop to Browse the Playlist
+Jockey3ME.TraxEncoderTurn = function (channel, control, value, status, group) {
+    var newValue = (value-64);
+
+   engine.setValue(group,"SelectTrackKnob",newValue);
+
+}
+
+// Browser Knop with Shift to Browse the Playlist Tree
+Jockey3ME.ShiftTraxEncoderTurn = function (channel, control, value, status, group) {
+    var newValue = (value-64);
+
+   if (newValue == 1) engine.setValue(group,"SelectNextPlaylist",newValue);
+   else engine.setValue(group,"SelectPrevPlaylist",1);
+}
+
+Jockey3ME.loop_double_halve = function (channel, control, value, status, group) {
+  var newValue = (value-64);
+
+  if (newValue == 1) {
+    engine.setValue(group,"loop_double",1);
+    engine.setValue(group,"loop_double",0);
+  } else {
+    engine.setValue(group,"loop_halve",1);
+    engine.setValue(group,"loop_halve",0);
+  }
+}
+
+Jockey3ME.move_beat = function (channel, control, value, status, group) {
+  var newValue = (value-64);
+  engine.setValue(group,"beatjump",(newValue*Jockey3ME.move_beat_value));
+}
+
+Jockey3ME.crossfader = function (channel, control, value, status, group) {
+  var newValue = 0;
+  if (control == 0x37 && !Jockey3ME.crossfaderScratch) {
+    newValue = ((value / 63.5) - 1);
+    engine.setValue(group,"crossfader",newValue);
+  } else if (control == 0x37 && Jockey3ME.crossfaderScratch) {
+    switch (value) {
+      case 127:
+        engine.setValue(group,"crossfader",1);
+        break;
+      case 126:
+        engine.setValue(group,"crossfader",0.96875);
+        break;
+      case 125:
+        engine.setValue(group,"crossfader",0.953125);
+        break;
+      case 2:
+        engine.setValue(group,"crossfader",-0.953125);
+        break;
+      case 1:
+        engine.setValue(group,"crossfader",-0.96875);
+        break;
+      case 0:
+        engine.setValue(group,"crossfader",-1);
+        break;
+      default:
+        engine.setValue(group,"crossfader",0);
+    }
+  } else {
+    if (value <= 126) {
+      script.crossfaderCurve(value, 0, 126);
+      Jockey3ME.crossfaderScratch = false;
+    } else {
+      Jockey3ME.crossfaderScratch = true;
+    }
+  }
+}
+
+Jockey3ME.CUP = function (channel, control, value, status, group) {
+  if (value == 0x7F) {
+    engine.setValue(group,"cue_default",1);
+    engine.setValue(group,"cue_default",0);
+    midi.sendShortMsg(status,control,127);
+  } else {
+    engine.setValue(group,"play",1);
+    midi.sendShortMsg(status,control,0);
+  }
+}
```

Comment:
Does this work with all cue modes? If you just want it to go to the cue point and play when the button is released, use the cue_gotoandplay control instead. This will ensure that the mapping works as intended even if the behavior of cue_default changes.